pages/am/3.x/user-guide/alerts/user-guide-alerts.adoc

```diff
@@ -25,6 +25,7 @@ The following table lists the available alert types.
 |===
 |Type|Key|Description
 | Too many login failures |too_many_login_failures| Alert when the number of login failures is abnormally high.
+| Risk-based alerts |risk_assessment| Alert when the user behaviour seem suspicious.
```

Comment:
maybe we can mark 'too many reset password', 'too many locked out... as not implemented ?

Reply:
FP